edi_oca/data/cron.xml

```diff
@@ -33,4 +33,20 @@
         <field name="state">code</field>
         <field name="code">model.search([])._cron_check_input_exchange_sync()</field>
     </record>
+    <record
+        id="cron_edi_backend_delete_superseded_records"
```

Comment:
```suggestion
        id="cron_edi_backend_delete_obsolete_records"
```
I think "obsolete" makes more sense than superseded. Can you apply the same find/replace everywhere?

Reply:
Thank @simahawk , I updated, could you have a look